src/autosuggest/interfaces.ts

```diff
@@ -80,6 +80,11 @@ export interface AutosuggestProps
    */
   enteredTextLabel?: AutosuggestProps.EnteredTextLabel;
 
+  /**
+   * Specifies whether to display the custom value indicator (for example, `Use "${value}"`).
```

Comment:
I am not sure if this line will be correctly presented in the website markdown.

How about we change the description and say that the value indicator is controlled by the `enteredTextLabel` property?

Reply:
Oh, nevermind - I just figured out this text is used in the enteredTextLabel already.

Still, it might be more clear if we say like: 
```
Hides the custom value indicator controlled with `enteredTextLabel` property
```

Reply:
Thanks, that wording is clearer, will update.